apps/dashboard/pages/api/v1/github/verify.ts

```diff
@@ -0,0 +1,196 @@
+import { Buffer } from "node:buffer";
+import crypto from "node:crypto";
+import { db } from "@/lib/db";
+import { env } from "@/lib/env";
+
+import type { Readable } from "node:stream";
+import { clerkClient } from "@clerk/nextjs";
+import { sha256 } from "@unkey/hash";
+import { Resend } from "@unkey/resend";
+import type { NextApiRequest, NextApiResponse } from "next";
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+  runtime: "nodejs",
+};
+const GITHUB_KEYS_URI = "https://api.github.com/meta/public_keys/secret_scanning";
```

Comment:
Maybe we should make the an env so if they decide to randomly change it we don't have to deploy.